src/themeStyles.ts

```diff
@@ -111,6 +111,21 @@ function themeToPillar(theme: Theme): string {
 	}
 }
 
+function themeToPillar(theme: Theme): Pillar {
+	switch (theme) {
```

Comment:
I think this is why you have the `Operands of '+' operation must either be both strings or both numbers.` error. This function used to return a string and now it returns an enum number.